decepticonlp/transforms/perturbations.py

```diff
@@ -309,9 +310,43 @@ def apply(self, word: str, **kwargs):
 
         char_array = np.array(list(word))
 
-        int_pick = np.random.randint(0, high=unicode_array.shape[0], size=len(word))
+        int_pick = np.random.choice(unicode_array.shape[0], len(word))
 
         picked_unicode = unicode_array[int_pick]
```

Comment:
This defeats the purpose of using np.random.choice. 

You can combine these two lines to use np.random.choice.